modules/doobie/src/test/scala/world/WorldData.scala

```diff
@@ -175,20 +175,21 @@ trait WorldMapping[F[_]] extends WorldPostgresSchema[F] {
       case Select("city", List(Binding("id", IntValue(id))), child) =>
         Select("city", Nil, Unique(Filter(Eql(UniquePath(List("id")), Const(id)), child))).rightIor
 
-      case Select("countries", List(Binding("limit", IntValue(num)), Binding("minPopulation", IntValue(min)), Binding("byPopulation", BooleanValue(byPop))), child) =>
+      case Select("countries", List(Binding("limit", IntValue(num)), Binding("offset", IntValue(offset)), Binding("minPopulation", IntValue(min)), Binding("byPopulation", BooleanValue(byPop))), child) =>
         def limit(query: Query): Query =
           if (num < 1) query
           else Limit(num, query)
 
-        def order(query: Query): Query =
+        def order(query: Query): Query = {
           if (byPop) OrderBy(OrderSelections(List(OrderSelection(UniquePath[Int](List("population"))))), query)
-          else query
+          else       OrderBy(OrderSelections(List(OrderSelection(UniquePath[String](List("code"))))), query)
+        }
```

Comment:
It's important to always have an explicit order if you're using OFFSET or LIMIT, otherwise the "paging" behavior won't really work. You get stuff back in arbitrary order.